docs/man/_index.md

```diff
@@ -26,6 +26,8 @@ command:
         - fatal
         - panic
       default: info
+    - name: with-access-token
+      description: access token for authentication via bearer token
```

Comment:
Is it a requirement that this access-token has been granted from the same client as the running platform's `public_client_id` client, or is it any AT from the idP running alongside the platform?

Reply:
I'm guessing the latter, but we might need to test and document in this manual either way.

Reply:
Validated locally with an access token granted from the client credentials flow (a client creds profile, `auth print-access-token`, and pasting the token into a `--with-access-token` request to list attributes).